docs/pages/billing/faq.mdx

```diff
@@ -0,0 +1,119 @@
+
+Use [EAS Update](/eas-update/introduction/) and [development builds](/develop/development-builds/introduction/) to test and deploy new code without creating a new build. This option is better for most apps since JavaScript code changes more frequently than the underlying native code. You can create multiple test channels with EAS Update and reduce the need to create additional builds for your team. For native code changes, you can also automate the build process using [Expo Fingerprint](https://expo.dev/blog/fingerprint-your-native-runtime).
+
+For more information, see [How to optimize build usage](/billing/usage-based-pricing/#how-to-optimize-build-usage).
+
+### How do I estimate my next bill?
+
+To estimate your next bill, go to [**Billing**](https://expo.dev/settings/billing). Under **Usage this month**, you will find a summary of your EAS Build usage based on the [resource class](/build/eas-json/#selecting-resource-class), EAS Update usage based on monthly updated users and global edge bandwidth, and the amount spent for both.
+
+See [How usage-based pricing works](/billing/usage-based-pricing/#how-usage-based-pricing-works) for more information.
+
+### What is a monthly updated user?
+
+A monthly updated user is a unique user who downloads an update via EAS Update in a single month, regardless of how many updates this user downloads in a given billing month/period. See [How are "monthly updated users" counted for a billing cycle](/eas-update/faq/#how-are-monthly-updated-users-counted-for-a-billing-cycle) for more information.
+
+## Payments
+
+### Can I pay annually?
+
+If you are on an [Enterprise plan](/billing/plans/#enterprise), [contact us](https://expo.dev/contact) for more information.
+
+### How can I update our payment information?
+
+To update your Organization account's payment information, make sure that you have either an [**Owner** or **Admin** role](/accounts/working-together/#manage-access). For a Personal account, you always have an **Owner** role. After verifying the role:
+
+- Go to your [**Account or Organization settings** > **Billing**](https://expo.dev/settings/billing), and click on **Update billing information**. This will take you to Stripe's portal.
+- On Stripe's portal, under **Payment method**, click on **Add payment method** to add a new payment method.
+
+See [Manage billing information](/billing/manage/#payment-method) for more details.
+
+### Can I pay with Automated Clearing House (ACH)?
+
+Only available for the [Enterprise plan](/billing/plans/#enterprise) who pay annually. [Contact us](https://expo.dev/contact) for more information.
+
+### I need a W-9, or other legal documentation
+
+For W-9, [contact us](https://expo.dev/contact).
+
+Find our legal terms at [expo.dev/terms](https://expo.dev/terms).
+
+### As a customer, can I get a Certification for US tax residency?
+
+This process takes time. [Contact us](https://expo.dev/contact) and our team will reach out.
```

Comment:
```suggestion
This process takes time. [Contact us](https://expo.dev/contact) and our team will reach out.
```
If we include this, I feel like we want to say this a little differently, because the length of time could be indefinite, as we depend on the IRS and they have no SLA for this. I wouldn't want to give the impression that there's a set wait time. Maybe we can workshop this more with Chris?

Reply:
I'll Chris about this specific FAQ.

Reply:
As per the latest discussion, I'm removing this from this version of the FAQ here.